part-time/code/4220-stefan-milkovski/src/app/recipe-list.ts

```diff
@@ -0,0 +1,1090 @@
+import { Recipe } from "./Recipe"
+
+export const RECIPE: Recipe = 
```

Comment:
No, the idea is not to have the data hard-coded in the application, it should be taken from the server